packages/grafana-data/src/field/overrides/processors.ts

```diff
@@ -67,8 +67,8 @@ export const valueMappingsOverrideProcessor = (
   value: any,
   _context: FieldOverrideContext,
   _settings?: ValueMappingFieldConfigSettings
-) => {
-  return value as ValueMapping[]; // !!!! likely not !!!!
+): ValueMapping[] => {
+  return value; // !!!! likely not !!!!
```

Comment:
lol

Reply:
Not sure how I feel about these.... at least before the smell was more obvious? Maybe it's fine because the value argument is still `any`?

I'm confused what this is supposed to do anyway?